BUILD

```diff
@@ -1265,6 +1265,73 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
```

Comment:
Please also update the `build.yaml` file and then run `./tools/buildgen/generate_projects.sh`.

Reply:
done

Reply:
Still need to run `./tools/buildgen/generate_projects.sh` to regenerate the other build files.